sharktank/sharktank/ops/default_impls.py

```diff
@@ -1264,3 +1288,23 @@ def zeros_like_default(
         requires_grad=requires_grad,
         memory_format=memory_format,
     )
+
+
+@ones_like.override(AllOfType(Tensor, PrimitiveTensor))
+def ones_like_default(
+    tensor: Union[Tensor, PrimitiveTensor],
+    *,
+    dtype: torch.dtype | None,
+    layout: torch.layout | None,
+    device: torch.device | None,
+    requires_grad: bool,
```

Comment:
Don't copy `kwargs` we are unlikely to use: `requires_grad` and `layout` are essentially irrelevant for our usecase